src/app/shared/components/checkbox-input/checkbox-input-field.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+import * as ReactForm from 'react-form';
```

Comment:
I think component should be renamed to`CheckboxField`.  Export CheckboxField from `shared/components/index.ts` so it can be imported as 

```
import { CheckboxField } from '../../../shared/components';
```